openapi/components/schemas/Organizations/OrganizationSettings.yaml

```
@@ -25,3 +25,21 @@ properties:
       Available only for US-based organizations.
     items:
       $ref: ./TaxLocation.yaml
+  notifications:
+    type: object
+    description: The settings for notifications.
+    properties:
+      notifyOnUserAccessChanges:
+        description: Specifies if a notification should be sent when there are changes in user access.
+        type: boolean
+        example: false
+      notifyOnApiKeyAccessChanges:
+        description: Specifies if a notification should be sent when there are changes in API key access.
+        type: boolean
+        example: false
+      notificationEmailAddresses:
+        description: A list of email addresses to send the notifications to.
```

nitpick: suggestion to avoid ending in a preposition

```suggestion
        description: List of email addresses where notifications are sent.
```